datalad/support/github_.py

```diff
@@ -185,13 +185,19 @@ def _gen_github_entity(
             yield ses.get_user(), token_str
 
 
-def _make_github_repos(
+def _make_github_repos_(
```

Comment:
just a note: I bet I would need to fixup crawler as well for any of the RFing here.  That logic is not tested in the crawler - no VCR tapes

Reply:
To me it looks like the crawler code only uses the credential/login pieces of this code and it should be unaffected.